src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -121,14 +122,56 @@ public Movie GetMovie(string title)
                 return _movieService.FindByTitle(title);
             }
 
-            var movies = _movieService.FindByTitle(parsedMovieInfo.MovieTitle, parsedMovieInfo.Year);
+            return LookupParsedMovie(parsedMovieInfo);
+        }
+
+        public Movie GetMovieFromPath(string path)
+        {
+            var parsedMovieInfo = Parser.ParseMoviePath(path, isLenient: false);
+
+            if (parsedMovieInfo != null)
+            {
+                return LookupParsedMovie(parsedMovieInfo);
+            }
 
-            if (movies == null)
+            return null;
+        }
+
+        private Movie LookupParsedMovie(ParsedMovieInfo parsedMovieInfo)
+        {
+            var movie = LookupParsedMovie(parsedMovieInfo, requireExactYear: true);
+
+            if (movie == null)
             {
-                movies = _movieService.FindByTitle(parsedMovieInfo.MovieTitle.Replace("DC", "").Trim());
```

Comment:
I looked into this random removal of "DC" and it comes from 52fb29ee189e7da807729683840097c58ef8ebc2 with extremely little explanation. My guess is this to do with edition information - should I move this into the Parser itself? I'm not sure how best to keep it in here.